src/data/oppfolgingstilfelle/person/oppfolgingstilfellePersonQueryHooks.ts

```diff
@@ -15,7 +15,13 @@ const sortByDescendingStart = (
   oppfolgingstilfelleList: OppfolgingstilfelleDTO[]
 ): OppfolgingstilfelleDTO[] => {
   return oppfolgingstilfelleList.sort((a, b) => {
-    return new Date(b.start).getTime() - new Date(a.start).getTime();
+    const startDateDifference =
```

Comment:
Er det mulig å trekke ut comparatoren i en egen variabel for lesbarhet? Feks sort(byEarliestAndLongestTilfelle) elns?